prow/external-plugins/needs-rebase/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-prow/git:0.1
```

Comment:
There is a 0.2 image available. We mind as well use the most current image.
https://github.com/kubernetes/test-infra/blob/2606fdca35d1aabb81b414c29c7cccf2a5da12a7/prow/Makefile#L21